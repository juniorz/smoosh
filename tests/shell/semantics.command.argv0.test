set -e

explicit=$(${TEST_UTIL}/argv)
[ "$explicit" = "argv[0] = \"${TEST_UTIL}/argv\";" ]

PATH="${TEST_UTIL}:$PATH"
inpath=$(argv)
[ "$inpath" = "argv[0] = \"argv\";" ]
argv

