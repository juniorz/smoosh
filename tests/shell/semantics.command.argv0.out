argv[0] = "argv";
